eureka/registry_test.go

```diff
@@ -185,13 +187,18 @@ var _ = Describe("OperateOnApplication", func() {
 				Expect(args.instanceId).To(Equal("instance-1"))
 			})
 
+			It("should log progress", func() {
+				Expect(progressWriter.String()).To(Equal("Processing service instance APP-1 with index 2\n"))
```

Comment:
The expected string needs to include the bold cyan formating of the instance id and the index otherwise the test fails.

Reply:
@georgeharley Fixed. I'll raise a story to explore making such tests fail under `govendor test +local`.